portal/integrated-windows-authentication/src/main/java/com/esri/samples/integrated_windows_authentication/IntegratedWindowsAuthenticationController.java

```diff
@@ -78,8 +78,8 @@ public void initialize() {
       resultsListView.setCellFactory(c -> new PortalItemInfoListCell());
 
       // hide the progress indicator when the map is finished drawing
-      mapView.addDrawStatusChangedListener(drawStatusChangedEvent -> {
-        if (drawStatusChangedEvent.getDrawStatus() == DrawStatus.COMPLETED) {
+      mapView.drawStatusProperty().addListener((observable, oldValue, newValue) -> {
+        if (newValue == DrawStatus.COMPLETED) {
```

Comment:
The comment on line 59 says `// show progress indicator while map is drawing` - if you remove the other instances of the progress indicator being set and bind the progress indicator to when drawing is in progress, is the behaviour the same?

Reply:
Behavior seems to be the same but I will double check this.

Reply:
Was not able to find a solution to replicate the indicator behavior when binding different properties using `Bindings.or`.